### DISCUSSION TITLE
Error 500 and action stuck on queue

### DISCUSSION THREAD

**Discussion by gnarlyeah** [community/community#47853] [Actions]
### Select Topic Area

Product Feedback

### Body

Having this error 500 problem and the action is stuck on queue status and it auto cancel the run, Any solution on this?

![image](https://user-images.githubusercontent.com/118649910/219830708-c183e018-0123-48a9-b4bf-b2cf5c86c3d5.png)

![image](https://user-images.githubusercontent.com/118649910/219830724-71fb76a9-e8fc-4358-aca9-450a384a7ccb.png)

**Comment by melvincwng** [comment]
Hey @gnarlyeah, there is an issue with the GitHub website.

The GitHub team is already looking into it. You can check the status updates over here: https://www.githubstatus.com/

Hope this clarifies and answers your question 😄

**Reply by melvincwng** [reply]
@gnarlyeah, the only solution right now is to kindly wait for GitHub engineers to update & fix the issue first, before handling any GitHub packages deployment or triggering of GitHub Actions.

**Reply by gnarlyeah** [reply]
Oh okay, I see. Thanks for the info.

**Reply by melvincwng** [reply]
No problem, glad to have helped 👍🏻

**Comment by LuminousEnergy** [comment]
This problem related to Github with many incident. i checked on github status

**Reply by gnarlyeah** [reply]
Hope this could get fix in no time. Thanks

**Reply by LuminousEnergy** [reply]
yeah its already more than 1 hours i cannot deploy

**Comment by melvincwng** [comment]
@gnarlyeah and @LuminousEnergy, try triggering another run of GitHub Actions.

Issue seems to be resolved now.

Tested a retrigger of GitHub Actions, and on my end, seems to be working fine now.

**Reply by gnarlyeah** [reply]
Unfortunately it didn't worked. But I deleted the repository and re uploaded it and deploy it. the action page now seems working I guess

**Reply by melvincwng** [reply]
Yup, seems like all of GitHub's systems are operational and working now.

You can always refer to this link: https://www.githubstatus.com/ to check the operational status of GitHub's systems.

Take care, and have a good day ahead 😃 

![20230218_163733](https://user-images.githubusercontent.com/77479885/219850728-14591c72-55a4-4004-95e4-79d5b21478a7.jpg)

**Reply by LuminousEnergy** [reply]
yes it's normal now, i can update the scripts

**Reply by melvincwng** [reply]
Glad to hear GitHub Actions is working for you 👍 @LuminousEnergy
